### PR TITLE
Add some missing strings to translations

### PR DESCRIPTION
**Pull request by tjw123hh**
Deleted some incorrect `notr="true"` tags to make the strings available in Weblate.

### CODE REVIEW

**Comment by TheKodeToad** [comment]
Can PermGen be translated?

**Comment by tjw123hh** [comment]
> Can PermGen be translated?

The colon after PermGen needs to be translated into full width(：) in some languages

**Comment by TheKodeToad** [comment]
Thanks GitHub! I guess it doesn't sign this off :sob:

**Comment by tjw123hh** [comment]
> Can PermGen be translated?

Besides, it seems to have translations (such as the “永久代” in Simplified Chinese)

File: program_info/org.prismlauncher.PrismLauncher.desktop.in
Comment (Scrumplex) [inline program_info/org.prismlauncher.PrismLauncher.desktop.in:1]: why did the permissions change here?

Comment (tjw123hh) [reply program_info/org.prismlauncher.PrismLauncher.desktop.in:1]: That was a misoperation

Comment (TheKodeToad) [reply program_info/org.prismlauncher.PrismLauncher.desktop.in:1]: I tried to Discard changes but that did literally nothing

Comment (TheKodeToad) [reply program_info/org.prismlauncher.PrismLauncher.desktop.in:1]: I fixed it. needed to force push. sorry!

**Review by Scrumplex — APPROVED** [review]

**Review by TheKodeToad — APPROVED** [review]

**Comment by Scrumplex** [comment]
Thanks!

**Pull request merged**